solr/core/src/test-files/solr/collection1/conf/schema15.xml

```diff
@@ -567,7 +567,17 @@
   <field name="_root_" type="string" indexed="true" stored="true"/>
   <!-- required for NestedUpdateProcessor -->
   <field name="_nest_parent_" type="string" indexed="true" stored="true"/>
-  <field name="_nest_path_" type="string" indexed="true" stored="true"/>
+  <field name="_nest_path_" type="descendants_path" indexed="true" multiValued="false" docValues="true" stored="false" useDocValuesAsStored="false"/>
+  <fieldType name="descendants_path" class="solr.SortableTextField">
+    <analyzer type="index">
+      <charFilter class="solr.PatternReplaceCharFilterFactory" pattern="(^.*.*$)" replacement="$0/"/>
```

Comment:
Can you please provide an example input String (here in GH) that has multiple levels and comment out it looks like when it's done?  I know how to read regexps but I need to stare at them a bit to figure them out, so lets make it easier to read/review.

Reply:
Sure thing